README.md

```diff
@@ -1,10 +1,14 @@
 [![CircleCI](https://circleci.com/gh/neo4j-contrib/neo4j-helm.svg?style=svg)](https://circleci.com/gh/neo4j-contrib/neo4j-helm)
 
 # DEPRECATED
-This project is deprecated and no further support will be provided.
-Please use the newly designed helm charts which support both Neo4j standalone and Neo4j cluster.
-You can find more info [here](https://github.com/neo4j-contrib/neo4j-helm.git) on the new helm charts
+Neo4j now publishes a productized and supported version of Helm Charts, which have been written using the experience generated by the neo4j-helm Labs project.
 
+For Neo4j standalone productized Helm charts are available for Neo4j 4.3 and above.
+For Neo4j Causal Cluster productized Helm charts are available for Neo4j 4.4 and above.
```

Comment:
This might be nit-picking but I think we either should remove the first `For` here or we should add a comma after `For Neo4j Causal Cluster`, same for Standalone.

Reply:
fixed